src/orquestra/sdk/__init__.py

```diff
@@ -54,6 +54,7 @@
     "TaskRun",
     "WorkflowDef",
     "WorkflowLogs",
+    "WorkflowLogTypeName",
```

Comment:
**non-blocking,suggestion**: this feels oddly specific, too specific for the main SDK public API. WDYT about making it a detail of `WorkflowLogs` so it would be seen by the user as `sdk.WorkflowLogs.TypeName` instead of `sdk.WorkflowLogsTypeName`?